### PR TITLE
Fixed not scrolling to top

### LINKED ISSUE #163: Community page remembers previous page scroll position
### Description
When I open the community page it remembers the previous page scroll position.
### Screenshots/testimonials
Capture

### PR DESCRIPTION
**Pull request by ankiiitraj**
### fixes #163 
<Do not delete this template>
Description: <provide a description> Added a HOC, that will execute scroll to top on every mounting.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes which were demanded in the linked issue.
- [x] I've tested my code on a Chromium based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

### CODE REVIEW

**Review by hariom1625 — APPROVED** [review]
LGTM !!

**Review by tusharjain0022 — APPROVED** [review]
LGTM 💯

File: client/src/components/HOC/withScrollToTop.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+
+const ScrollToTop = (Component) => {
+    const NewComponent = () => {
+        useEffect(() => {
+            window.scrollTo(0, 0);
+        }, []);
+        return(
+            <Component />
```
Comment (EmperorYP7) [inline client/src/components/HOC/withScrollToTop.jsx:9]: Learnt something from this. Looks great @ankiiitraj! 🤩

**Review by EmperorYP7 — APPROVED** [review]

**Comment by EmperorYP7** [comment]
Merging this.

**Pull request merged**